_maps/map_files/RandomRuins/SpaceRuins/meatpackers.dmm

```diff
@@ -20,7 +20,6 @@
 /area/ruin/unpowered/BMPship/Delta)
 "af" = (
 /obj/machinery/porta_turret{
-	installation = /obj/item/gun/energy/gun;
 	lethal = 1;
 	name = "ship defense turret"
 	},
```

Comment:
Could you pull all of this into a custom turret subtype, e.g.
```
/obj/machinery/porta_turret/meatpacker_ship
	name = "ship defense turret"
	lethal = 1
```

And then swap these instances out with that?